src/External/CMakeLists.txt

```diff
@@ -190,4 +199,5 @@ list(APPEND EXTERNAL_LIBRARIES
      ${tinyfiledialogs_LIBRARIES})
 
 set(EXTERNAL_INCLUDE_DIRS ${EXTERNAL_INCLUDE_DIRS} PARENT_SCOPE)
+set(EXTERNAL_LIBRARY_DIRS ${EXTERNAL_LIBRARY_DIRS} PARENT_SCOPE)
 set(EXTERNAL_LIBRARIES    ${EXTERNAL_LIBRARIES}    PARENT_SCOPE)
```

Comment:
Do we need to maintain both ``EXTERNAL_LIBRARY_DIRS`` and ``EXTERNAL_LIBRARIES``?

Reply:
I think we need. ``EXTERNAL_LIBRARIES`` are the raw library names. ``EXTERNAL_LIBRARY_DIRS`` are the search directories. Some compilers are not smart enough to search the OS to find the libraries (Visual Studio, Xcode, etc), so both have to be specified in CMakeLists.